dev/MRTCore/mrt/Microsoft.ApplicationModel.Resources/unittests/MrtCoreManagedTest.csproj

```diff
@@ -153,6 +154,9 @@
     </PackageReference>
   </ItemGroup>
   <ItemGroup>
+    <None Include="MrtCoreManagedTests.runsettings">
```

Comment:
><None Include="MrtCoreManagedTests.runsettings"> [](http://example.com/codeflow?start=4&length=48)

what makes this a new requirement?

Reply:
Same as for MrtCoreUnpackagedTests. Removing that file and the powershell postbuildevent to update its path results in "Test run aborted"

MSTest doesn't run the test without it.

Reply:
strange.